src/python/foglamp/configuration_manager.py

```diff
@@ -56,29 +60,37 @@
                 value_val - string (dynamic)
 """
 
+def _run_callbacks(category_name):
+    callbacks = _registered_interests.get(category_name)
+    if callbacks is not None:
+        for callback in callbacks:
+            cb = import_module(callback)
+            cb.run()
```

Comment:
It might be a good idea to pass the category in the callback, that way a single callback can be shared between multiple categories if a component had more than one category it was interested in.